src/clinical/api/clinical-api.ts

```diff
@@ -140,6 +143,40 @@ class ClinicalController {
     res.send(zip.toBuffer());
   }
 
+  /**
+   * Download Clinical Data selected by Donor ID.
+   * Requires Full Read Access.
+   * Used by API for a user to download clinical data for specific donors and/or files.
+   * @param req
+   * @param res
+   * @returns
+   */
+  @HasFullReadAccess()
+  async getDonorDataByIdAsTsvsInZip(req: Request, res: Response) {
+    const bodyParseResult = DonorDataApiBody.safeParse(req.body);
+    if (!bodyParseResult.success) {
+      return ControllerUtils.badRequest(
+        res,
+        `Invalid data in request body: ${JSON.stringify(bodyParseResult.error)}`,
+      );
+    }
+    const { donorIds } = bodyParseResult.data;
+
+    const donorEntityData = await service.getDonorEntityData(donorIds);
+
+    const date = currentDateFormatted();
+    const fileName = `filename=Donor_Clinical_Data_${date}.zip`;
```

Comment:
This is sharing the naming convention used for for downloads requested by program submitters. I'm open to suggestions to improve the provided filename. Perhaps a timestamp is a good idea in addition to the current date.

Reply:
> aside from the filename format itself, would it make sense to abstract this into an env variable?

I don't see a reason to have different filenames per env, what's your reason? If you want filename constanta centralized into a constants file I can do that.

Reply:
Tbh, I'm just used to abstracting constants like this one, because that makes it easier for these values to be updated, without having to rebuild the code "just for a string change". 

If you believe this doesn't need to be abstracted, I of course trust your insight.

Reply:
Good reason. will make the change.